doc/en/user/source/extensions/netcdf/netcdf.rst

```diff
@@ -325,45 +450,81 @@ For example, it could be a ``netcdf_index.properties`` file with the following c
     preparedStatements=true
     max\ connections=20
 
-Then, in order to migrate a specific mosaic, run the **H2Migrate** tool::
+Notice how the NetCDF indexes are going to be stored in a different database schema, to prevent
+eventual table name conflicts (again, make sure the schema already exists in the database).
+
```

Comment:
We could consider also suggesting to use the storeName as we do in the Migration chapter.
So that the datastore is being provided by the GeoServer catalog repository.

Reply:
Eh... I actually wanted to avoid it, to keep things simple. I'd rather link the documentation for the property files (hopefully storename is in there) and remove it from the H2Migrate part.